bigquery/tables.rb

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def create_table project_id:, dataset_id:, table_id:
+def create_table_with_schema project_id:, dataset_id:, table_id:
   # [START create_table]
```

Comment:
Per our conversation yesterday, you're going to update the region tags in a separate PR, right?

Reply:
Yeah it currently doesn't have a branch specified, so changing the region tag now would break the link